datatypes/src/jvmTest/kotlin/EcCurveTest.kt

```diff
@@ -0,0 +1,31 @@
+import at.asitplus.crypto.datatypes.EcCurve
+import com.ionspin.kotlin.bignum.integer.toBigInteger
+import io.kotest.core.spec.style.FreeSpec
+import io.kotest.matchers.shouldBe
+
+class EcCurveTest: FreeSpec({
+    "SECP256 modulus correct" {
+        EcCurve.SECP_256_R_1.modulus shouldBe
+                (2.toBigInteger().shl(223)
```

Comment:
I'll guess the values are copied here from somewhere, but we should document that.